### PR TITLE
changes for adding properties to callog to track rac timestamp value

### PR DESCRIPTION
**Pull request by rasamala83**

### CODE REVIEW

**Comment by rasamala83** [comment]
This change adds an extra 80 to 95 bytes for every RAC_ID cal event. 

Sample:
E18:17:19.76	RAC_ID	0	0	poolModName=hera-test&workerId=0&shardId=0&tm=1712926025&exitTime=1712926087&exitInSec=48sec
E18:17:19.76	RAC_ID	0	0	poolModName=hera-test&workerId=0&shardId=0&tm=1712926025&exitTime=1712926135&exitInSec=96sec
E18:17:19.76	RAC_ID	0	0	poolModName=hera-test&workerId=0&shardId=0&tm=1712926025&exitTime=1712926183&exitInSec=144sec
E18:17:19.76	RAC_ID	0	0	poolModName=hera-test&workerId=0&shardId=0&tm=1712926025&exitTime=1712926231&exitInSec=192sec

File: lib/workerpool.go [hunk 1]
```diff
@@ -727,8 +727,19 @@ func (pool *WorkerPool) RacMaint(racReq racAct) {
 	pool.poolCond.L.Unlock()
 	// TODO: C++ worker logs one event for each worker, in the worker, so
 	// we keep the same. Think about changing it
-	for i := 0; i < cnt; i++ {
+	for i := 0; i < len(racMaintWorkers); i++ {
+		poolModName, _ := racMaintWorkers[i][0].(string)
+		shardId, _ := racMaintWorkers[i][1].(int64)
+		workerId, _ := racMaintWorkers[i][2].(int64)
```
Comment (venkatsridhar95) [inline lib/workerpool.go:733]: Can we also track pid?

File: lib/workerpool.go [hunk 2]
```diff
@@ -718,7 +718,7 @@ func (pool *WorkerPool) RacMaint(racReq racAct) {
 			if logger.GetLogger().V(logger.Verbose) {
 				logger.GetLogger().Log(logger.Verbose, "Rac maint activating, worker", i, pool.workers[i].pid, "exittime=", pool.workers[i].exitTime, now, window, pool.currentSize)
 			}
-			cnt++
+			racMaintWorkers = append(racMaintWorkers, []interface{}{pool.moduleName, pool.ShardID, i, pool.workers[i].exitTime, pool.workers[i].exitTime - now})
```
Comment (venkatsridhar95) [inline lib/workerpool.go:721]: Change looks good to me. Instead of this, can we have an event inside this loop?

File: lib/workerpool.go (hunk 1 above)
Comment (rasamala83) [reply lib/workerpool.go]: added pid to CAL event

E16:18:21.36	RAC_ID	0	0	poolModName=hera-test&workerId=1&pid=83228&shardId=0&tm=1714992488&exitTime=1714992541&exitInSec=40sec
E16:18:21.36	DB_UNAME	 MyDB	0	
E16:18:21.36	RAC_ID	0	0	poolModName=hera-test&workerId=2&pid=83227&shardId=0&tm=1714992488&exitTime=1714992581&exitInSec=80sec
E16:18:21.37	DB_UNAME	 MyDB	0	
E16:18:21.37	RAC_ID	0	0	poolModName=hera-test&workerId=3&pid=83232&shardId=0&tm=1714992488&exitTime=1714992621&exitInSec=120sec
E16:18:21.37	DB_UNAME	 MyDB	0	
E16:18:21.37	RAC_ID	0	0	poolModName=hera-test&workerId=4&pid=83231&shardId=0&tm=1714992488&exitTime=1714992661&exitInSec=160sec
E16:18:21.37	DB_UNAME	 MyDB	0	
E16:18:21.37	RAC_ID	0	0	poolModName=hera-test&workerId=5&pid=83229&shardId=0&tm=1714992488&exitTime=1714992701&exitInSec=200sec

**Review by ModEtchFill — APPROVED** [review]
Probably not possible to add automated tests since RAC isn't in the express edition that github actions workflows tests on. Approving on my end to avoid blocking your internal steps.

**Pull request merged**